### PR TITLE
Change zenml init --template to optionally prompt and track email

### PR DESCRIPTION
**Pull request by stefannica**
## Describe changes
Add a `--starter` flag to `zenml init` to generate the starter template without prompting for user input (use default param values).
Both `zenml init --starter` and `zenml init --template` ask for a user email the first time they are used, which is tracked.

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/component-gallery/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: src/zenml/cli/base.py [hunk 1]
```diff
@@ -54,40 +61,96 @@
...
-            "Next, you will be prompted to generate a project from the "
-            "template."
-        )
-        CopierApp.run(
-            ["copier", "gh:zenml-io/zenml-project-templates", str(path)],
-            exit=False,
+
+        from zenml.cli.text_utils import (
+            zenml_go_privacy_message,
+            zenml_go_welcome_message,
```
Comment (strickvl) [inline src/zenml/cli/base.py:106]:
```suggestion
            zenml_cli_privacy_message,
            zenml_cli_welcome_message,
```

Probably wants changing? (and in `text_utils.py` file)

**Review by strickvl — CHANGES_REQUESTED** [review]
Core functionality is fine, but when you actually do this it looks a bit weird in the CLI:

ScreenShot 2023-02-07 at 17 41 36

Esp the 'Next you will be prompted..." line. Either could do with being broken up by some line breaks, or a header, or something to separate it from the privacy statement above.

**Comment by stefannica** [comment]
Better, I hope: 

Screenshot from 2023-02-07 18-20-44

**Review by strickvl — APPROVED** [review]
Looks good now!

**Pull request merged**